n2kMappings.js

```diff
@@ -12,6 +12,39 @@ function getMmsiContext(n2k) {
 }
 
 exports.mappings = {
+  //Battery Voltage
+  '127508': [{
+    source: 'Voltage',
+    node: 'electrical.batteries.0.capacity.stateOfCharge',
+    filter: function(n2k) {
+      return n2k.fields['Battery Instance'] === 0;
+    }
+  }, 
+{
+    source: 'Voltage',
+    node: 'electrical.batteries.1.capacity.stateOfCharge',
```

Comment:
Got it. What about the "voltage" issue? From what I understand, It's very difficult to compute "stateOfCharge" for lead acid batteries. Can you add voltage to the spec?